### DISCUSSION TITLE
The Prometheus Operator version used by Kafka Operator

### DISCUSSION THREAD

**Discussion by AirTrioa** [strimzi/strimzi-kafka-operator#9440] [Q&A]
- Kafka Operator comes with a Prometheus, which I found to be deployed by the Prometheus Operator（ https://github.com/strimzi/strimzi-kafka-operator/blob/release-0.27.x/examples/metrics/prometheus-install/prometheus.yaml ）But when I added some configurations, I found that the configurations were not supported. 
- I would like to know which version of Kafka Operator is used in version 0.27.1?
![image](https://github.com/strimzi/strimzi-kafka-operator/assets/44222967/ece4f366-1e64-4d3e-98f0-6969d1d7fd57)

**Comment by scholzj** [comment]
> I would like to know which version of Kafka Operator is used in version 0.27.1?

I'm not sure what do you mean with this point. Strimzi 0.27.1 uses Strimzi 0.27.1.

> Kafka Operator comes with a Prometheus, which I found to be deployed by the Prometheus Operator（https://github.com/strimzi/strimzi-kafka-operator/blob/release-0.27.x/examples/metrics/prometheus-install/prometheus.yaml). But when I added some configurations, I found that the configurations were not supported.

0.27.1 is almost 2 years old. You probably shouldn't be using it and should upgrade. But leaving that aside -> the Prometheus installation is just an example. You can use any Prometheus you have and install it any way you want. You just have to configure it to scrape the Strimzi metrics endpoint. That is it. So you are not in any way bound to the version of the Prometheus operator used in 0.27.1 examples. You can use any Prometheus you want, you just have to configure it your self.
